data/hai/unfettered 0 prologue.txt

```diff
@@ -297,6 +297,133 @@ mission "Unfettered Jump Drive 3"
+				`	(Try to dodge it.)`
+					goto dodge
+				`	(Leap to the side and use my own weapon.)`
+				`	(Flee.)`
+					goto flee
+			`	You leap to the side, grabbing your own weapon just in time to threaten the Unfettered that threw the blade at you. Seeming a bit surprised by your unconventional tactics, they lower their own weapon. "Well done! In truth, there is no higher honor than surviving on the field of battle, because only then can you bring loot back to those who need it! A tactic is only good if you can find and execute it quickly enough to give you an edge, which you did. I am Desneeparu, and it is my pleasure to meet you, Captain."`
+				goto meet
+			label grab
+			`	You manage to grab the weapon, just in time to deflect an incoming attack from the same Unfettered. They duel with you for what feels like an eternity. At first you are astonished by the fact you have managed to survive this long, but you soon realize that, as you have not managed to land a single successful blow, this must be some kind of deliberate test.`
+			`	Then, suddenly, you can feel the pace of your opponent change; the game is over. They strike a masterful blow that disarms you with ease before smiling and lowering their weapon. "It looks like you know what to do with a blade! I am Desneeparu, and it is a pleasure to meet you, Captain <last>!" she exclaims, before laughing at your surprised face. You realize she must have been toying with you from the beginning, but, nevertheless, she seems satisfied by your performance.`
+			label meet
+			action
+				log "minor people" "Desneeparu" `An Unfettered warrior that knows how to handle a blade. She fought you to test your resolve before you were given access to their weapons. You suspect she was holding back, or you would never have stood a chance.`
+			`	Desneeparu is a feminine name, and she clearly knows how to handle her weaponry. She does some kind of salute with her fist before leaving. "I most go now. I hope you will find the weapon you need, Captain."`
+			`	The engineer, who had stepped back during the fight, now proceeds to take you underground to what looks like a very basic weapon testing range. Many Hai are there, experimenting with small modifications, but never performing any extreme tests. Seeing your interest, the engineer explains, "We prefer to do most of our testing on the field of battle. Here we only make sure the weapon actually fires before Unfettered captains can get their hands on it." She points to where a few captains watch the tests; apparently satisfied with what they see, one even requests a weapon to be taken to their ship. You keep walking until you enter the outfitter section of the facility, where you notice the items on display are much more advanced than anything you have seen available on the surface: so much so it could almost pass for a 'normal' Hai outfitter, were it not for the overcharged, oppressive feel of the air, which is probably due to extensive weapon testing.`
```

Comment:
```suggestion
			`	The engineer, who had stepped back during the fight, now proceeds to take you underground to what looks like a very basic weapon testing range. Many Hai are there, experimenting with small modifications, but never performing any extreme tests. Seeing your interest, the engineer explains. "We prefer to do most of our testing on the field of battle. Here we only make sure the weapon actually fires before Unfettered captains can get their hands on it." She points to where a few captains watch the tests; apparently satisfied with what they see, one even requests a weapon to be taken to their ship. You keep walking until you enter the outfitter section of the facility, where you notice the items on display are much more advanced than anything you have seen available on the surface: so much so it could almost pass for a 'normal' Hai outfitter, were it not for the overcharged, oppressive feel of the testing grounds.`
```